src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/Interceptingelement.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System;
+using System.ServiceModel.Channels;
+using System.ServiceModel.Configuration;
+
+namespace Microsoft.Samples.MessageInterceptor
+{
+    /// <summary>
+    /// Configuration class for InterceptingBindingElement. To make your InterceptingBindingElement
+    /// configurable, derive from InterceptingElementInterceptingElement and override CreateMessageInterceptor()
+    /// </summary>
+    public abstract class InterceptingElement : BindingElementExtensionElement
```

Comment:
Nit: the filename casing does not match the class name (Interceptingelement.cs).